bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -149,6 +149,9 @@ void ReplicaImp::onReportAboutInvalidMessage(MessageBase *msg, const char *reaso
 
 template <>
 void ReplicaImp::onMessage<ClientRequestMsg>(ClientRequestMsg *m) {
+  if (isSeqNumToStopAt(lastExecutedSeqNum))
```

Comment:
It might be useful to add a log message explaining why the message is ignored.
The same for `PrePrepareMsg`.

WDYT?

Reply:
agreed. I will add one